infra/base-images/base-runner/bad_build_check

```diff
@@ -203,7 +209,12 @@ function check_mixed_sanitizers {
   local CALL_INSN=
   case $(uname -m) in
       x86_64)
+        if [[ $ARCHITECTURE == 'i386' ]]
```

Comment:
Shouldn't this be outside the case.

if [[ $ARCHITECTURE == 'i386' ]]
then 
   CALL_INSN="call\s+[0-9a-f]+\s+<"
else
   case $(uname -m) in
      x86_64)	      x86_64)
....

it seems weird to put i386 in a x86_84 branch.

Reply:
Done.
Side note: I find this case statement a bit weird to begin with. It assumes that we will be compiling for arm on an arm machine. I think cross compilation is the more likely scenario.